genienlp/models/base.py

```diff
@@ -125,6 +125,24 @@ def __init__(
 
 # TransformerSeq2Seq and TransformerLSTM will inherit from this model
 class GenieModelForGeneration(GenieModel):
+    def numericalize_example(self, input_text, turn_id, device):
+        if isinstance(input_text, str):
+            input_text = [input_text]
+        tokenized_contexts = self.numericalizer.encode_batch(input_text, field_name='context', features=None)[0]
+
+        numericalized_turn = NumericalizedExamples(
+            example_id=[str(turn_id)],
+            context=SequentialField(
```

Comment:
Is there a reason we don't just use `tokenized_contexts` instead of creating a new `SequentialField`?

Reply:
`tokenized_contexts` attributes have list type. We need to turn them into tensors before passing it to the model.